app/vue/src/client/preview/story_store.js

```diff
@@ -0,0 +1,89 @@
+/* eslint no-underscore-dangle: 0 */
+
+let count = 0;
+
+function getId() {
+  count += 1;
+  return count;
+}
+
+export default class StoryStore {
```

Comment:
Isn't this duplicated from `@storybook/react`? If this is the core of our front-end, can we reuse it somehow?